packages/ui-components/src/components/Table/ExpandableTableBody.js

```diff
@@ -17,8 +17,9 @@ const StyledTableBody = styled(TableBody)`
 `;
 
 export const ExpandableTableBody = ({ ExpandableTableRow, SubComponent, ...tableBodyProps }) => {
-  const TableRow = tableRowProps => (
-    <ExpandableTableRow {...tableRowProps} SubComponent={SubComponent} />
+  const TableRow = React.useCallback(
```

Comment:
wrap the component in useCallback to prevent it being re-mounted on every render